### ISSUE TITLE
./configure --enable-option-checking uses unknown options

### ISSUE THREAD

**Issue by janstary** [erikd/libsamplerate#30]
This is 0.1.9 on MacOS 10.13.2.
Turning on `--enable-option-checking` as the only arg to `./configure` says:

```
/Users/hans/src/libsamplerate-0.1.9/Cfg/missing: Unknown `--is-lightweight' option
Try `/Users/hans/src/libsamplerate-0.1.9/Cfg/missing --help' for more information
configure: WARNING: 'missing' script is too old or missing
```

Also, it continues to run, instead of exiting on an unknown option.

**Comment by erikd** [comment]
No such warning on Linux. Have you tried re-running `./autogen.sh` ?

If that still doesn't fix it, please provide the versions of autoconf, automake, libtool and pkg-config you are using.

**Comment by janstary** [comment]
No such woraning on MacOS or OpenBSD eithet, with current git.
Perhaps this has been fixed since 0.1.9

**Comment by erikd** [comment]
Still not seeing this on Linux with autoconf version 2.69. Closing this.
